### ISSUE TITLE
Add node.js support in the container

### ISSUE THREAD

**Issue by piotrwitek** [make-software/casper-nctl-docker#13]
**What happened?**
We want to use this docker container in automated tests for js-clients @davidatwhiletrue 
The actual LTS version is preferred.

**What did you expect to happen?**
Node.JS is preinstalled in the docker container

**Are you willing to submit a pull request to fix this bug?**
There is no Node.JS in the docker container

**Comment by davidatwhiletrue** [comment]
Thanks @piotrwitek for this suggestion. After a bit of analysis we think the image must remain without NodeJS or other runtimes not needed for the NCTL service itself.
For Continuous Integration tasks a Docker Compose file could be used to have a NodeJS container in parallel to NCT. Also with GitHub actions someone can install the required dependencies to run integration testings. For who might be interested in this approach, see [here](https://github.com/make-software/casper-net-sdk/blob/master/.github/workflows/integration-testing.yml).
